### PR TITLE
NAS-113941 / 24.04 / Sync email validation rules WebUI/Middleware

### PR DESCRIPTION
**Pull request by bvasilenko**

### CODE REVIEW

**Review by denysbutenko — CHANGES_REQUESTED** [review]
Let's add unit tests for new validator.

**Review by denysbutenko — COMMENTED** [review]
However, it works well while basic checking. Consider of replacing builtin validator with new one for other components.

**Comment by denysbutenko** [comment]
As final point would be good to prevent using built-in validator in the future. Take a look on eslint config, specifically at `no-restricted-syntax` that allows add linter rule for this case.

**Comment by bvasilenko** [comment]
Ok, requested changes should be addressed now

File: src/app/modules/ix-forms/validators/email-validation/email-validation.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+describe('emailValidator', () => {
+  const validate = emailValidator();
+
+  it('should not return an error when value is valid email', () => {
+    expect(validate(new FormControl('test@mail.com'))).toBeNull();
+    expect(validate(new FormControl('a@a.a'))).toBeNull();
+  });
+
+  it('should return an error when value is valid email', () => {
```
Comment (undsoft) [inline src/app/modules/ix-forms/validators/email-validation/email-validation.spec.ts:12]: when value is invalid email.

**Review by undsoft — APPROVED** [review]
Looks good.

**Review by denysbutenko — APPROVED** [review]
Looks good. Thanks for adding tests and linter rule.

**Pull request merged**